core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -26,132 +26,180 @@ THE SOFTWARE.
     <d:taglib uri="local">
-        <d:tag name="revokeIcon">
-          <l:icon src="symbol-trash" />
+        <d:tag name="editButton">
+            <button type="button"
+                    class="jenkins-table-button jenkins-button jenkins-button--tertiary api-token-property-token-rename"
+                    data-target-url="${descriptor.descriptorFullUrl}/rename" data-rename-title="${%RenameToken}"
+                    data-rename-message="${%New name}"
+                    tooltip="${%Rename this token}">
+                <l:icon src="symbol-edit"/>
+            </button>
+        </d:tag>
+        <d:tag name="revokeButton">
+            <button type="button"
+                    class="jenkins-table-button jenkins-button jenkins-button--tertiary jenkins-!-destructive-color api-token-property-token-revoke"
+                    data-message-if-legacy-revoked="${%RevokedToken}"
+                    data-confirm="${%ConfirmRevokeSingle}"
+                    data-confirm-title="${%ConfirmRevokeSingleTitle}"
+                    data-token-uuid="${token.uuid}"
+                    data-target-url="${descriptor.descriptorFullUrl}/revoke" tooltip="${%RevokeToken}">
+                <l:icon src="symbol-trash" />
+            </button>
         </d:tag>
     </d:taglib>
+    <template id="api-token-row-template" xmlns:local="local">
+        <tr>
+            <td>
+                <div class="token-name-container">
+                    <span class="token-name"/>
+                </div>
+            </td>
+            <td>
+                <span class="token-creation-date" tooltip="${%Just now}">${%Today}</span>
+            </td>
+            <td align="right" class="token-use-counter">0</td>
+            <td align="right">
+                <div class="token-last-used">${%Never}</div>
+            </td>
+            <td class="jenkins-table__cell--tight">
+                <div class="jenkins-table__cell__button-wrapper">
+                    <local:editButton/>
+                </div>
+            </td>
+            <td class="jenkins-table__cell--tight">
+                <div class="jenkins-table__cell__button-wrapper">
+                    <local:revokeButton/>
+                </div>
+            </td>
+        </tr>
+    </template>
+    <div class="jenkins-hidden" id="api-token-template">
+        <div class="api-token-form">
+            ${%Name}: <span class="api-token-new-name"/>
+            <div class="api-token-new-form-row">
+                <span>${%Token}:</span>
+                <span class="api-token-new-value"/>
+                <l:copyButton text="" clazz="jenkins-hidden" tooltip="${%CopyToken}" iconOnly="true"/>
+            </div>
+            <div class="api-token-new-warning">
+                <l:icon src="symbol-warning" class="icon-md jenkins-!-warning-color"/>
+                ${%TokenDisplayedOnce}
```

Comment:
I think an alert-into above the token would look better and not be so distracting: https://weekly.ci.jenkins.io/design-library/banner/

Reply:
![image](https://github.com/user-attachments/assets/548d5176-18ab-45cf-b4c4-42323b628e4a)

Reply:
left align the content and try changing from warning to info?

Reply:
![image](https://github.com/user-attachments/assets/a3c1e225-849e-4c45-b242-06e5d32c5b03)

Reply:
I think this is better (assuming the dialog stays)

Reply:
Could we use `jenkins-quote jenkins-quote--monospace` for the token? It's currently in use on the System page + Jenkins CLI - would draw more attention to it.

<img width="703" alt="image" src="https://github.com/user-attachments/assets/905aa401-506e-4db3-87dd-916f303826bf" />